lib/ReactViews/NowViewing/Voldemort.jsx

```diff
@@ -0,0 +1,76 @@
+'use strict';
+
+import ObserveModelMixin from './../ObserveModelMixin';
+import React from 'react';
+
+const Voldemort = React.createClass({
+    mixins: [ObserveModelMixin],
+
+    propTypes: {
+        nowViewingItem: React.PropTypes.object
+    },
+
+    toggleOpen(item) {
+        item.isOpen = !item.isOpen;
+    },
+
+    toggleActive(item) {
+        item.isActive = !item.isActive;
```

Comment:
Don't set active items this way - selecting one doesn't unselect the old one, even when only one can be selected. Use `item.toggleActive()`.